godot-codegen/src/models/domain.rs

```diff
@@ -314,6 +315,9 @@ pub trait Function: fmt::Display {
     fn direction(&self) -> FnDirection {
         self.common().direction
     }
+    fn is_required(&self) -> bool {
+        self.common().is_required
+    }
```

Comment:
```suggestion

    fn is_virtual_required(&self) -> bool {
        self.common().is_virtual_required
    }
```
Note the empty line.